### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by ramitos**
fixes yldio/joyent-portal#1177

### CODE REVIEW

File: _env.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const { homedir } = require('os');
```
Comment (cjihrig) [inline _env.js]: Instead of having this file, could you just add the `SDC_KEY_PATH` default value logic in index.js?

Comment (ramitos) [reply _env.js]: yeah, sure

File: index.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 'use strict';
+
 const Brule = require('brule');
 const Hapi = require('hapi');
 const Rollover = require('rollover');
+const { homedir } = require('os');
+const { join } = require('path');
+
+process.env.SDC_KEY_PATH =
+  process.env.SDC_KEY_PATH || join(homedir(), './.ssh/id_rsa');
```
Comment (cjihrig) [inline index.js]: The `./` before `.ssh` shouldn't be needed.

Comment (ramitos) [reply index.js]: you are right, I don't know how I missed that 🤦‍♂️

**Pull request merged**